src/components/modal/Modal.scss

```diff
@@ -144,25 +146,50 @@
             padding: 0;
         }
 
+        .modal-header-container {
+            position: absolute;
+            top: 0;
+            left: 0;
+            display: flex;
+            align-items: center;
+            width: 100%;
+            height: $responsive-modal-header-height;
+            background-color: $white;
+        }
+
+        .modal-header {
+            display: flex;
+            align-items: center;
+            padding: 0 16px;
+        }
+
         .modal-content {
-            padding-bottom: 60px;
+            margin-top: $responsive-modal-header-height;
+            margin-bottom: $responsive-modal-actions-height;
+            padding: 24px 16px 12px 16px;
+            overflow: auto;
         }
 
         .modal-dialog {
+            height: 100%;
             min-height: 100%;
-            padding: 22px 16px 12px 16px;
+            padding: 0;
             border-radius: 0;
         }
 
         .modal-close-button {
+            top: 2px; //for visual alignment
```

Comment:
What are you thoughts on adding `display: flex` and `align-items: center` with `top: 0`? I think this could avoid the pixel adjustment for visual alignment.

Reply:
### With adding `flex` and `top: 0`
![Screen Shot 2022-07-12 at 11 01 17 PM](https://user-images.githubusercontent.com/6400298/178661925-763f8002-1abe-492b-877d-e1b742e81946.png)

### With `top: 2px`
![Screen Shot 2022-07-12 at 11 02 49 PM](https://user-images.githubusercontent.com/6400298/178661941-bee822fc-29ad-4e08-9772-70045188c03e.png)

Seems like it's still slightly off. However, I'm wondering if I'm optimizing at the 1% level. I'm open to using the `flex` version. It's probably better for long term maintainability. I've made the change in the update, but let me know what you think.

Reply:
Ahh interesting, I didn't expect there to be differences. In this case, go with your best judgement and feel free to change it back to `top: 2px` if it works better.

I'm curious how it'll display on Windows computers. If you have Parallels, this could be a good test to help sway your decision one way or the another as it's likely to be similarly misaligned by a couple pixels. I've noticed that our 'Lato' font is rendered differently by OS environment and has additional bottom padding on Windows computers.

For example:

**Chrome on macOS**
<img width="240" alt="Screen Shot 2022-07-13 at 10 17 06 AM" src="https://user-images.githubusercontent.com/7311041/178793299-6397ee84-63ac-4b19-9869-422513d3b6f4.png">

**Chrome on Windows**
<img width="240" alt="Screen Shot 2022-07-13 at 10 16 39 AM" src="https://user-images.githubusercontent.com/7311041/178793384-0e879ea8-1bd0-4311-b489-fa803f6befc5.png">